### PR TITLE
omnisci image fix

### PR DESCRIPTION
**Pull request by dchigarev**
#2083 fix omnisci image version to pass CI

### CODE REVIEW

**Comment by xmnlab** [comment]
mmmm sorry ... I just realized that #2083 is not using the tag `d7e34d4` (that is used currently) ... that PR is changing to `a7b9a02` .. and `a7b9a02` is not available.

so just using the tag that is currently used on `master` should fix that problem.

but if you want to update to `8339f67` go ahead ;)

**Comment by dchigarev** [comment]
I'm continuing work at #2083 for @semelianova,
that's why i brought her commits at this branch

File: ci/docker-compose.yml [hunk 1]
```diff
@@ -89,7 +89,7 @@ services:
       KUDU_MASTER: "false"
 
   omniscidb:
-    image: omnisci/core-os-cpu-dev:d7e34d4
+    image: omnisci/core-os-cpu-dev:cecceef
```
Comment (anmyachev) [inline ci/docker-compose.yml:92]: why you changed this?

**Comment by jreback** [comment]
thanks @dchigarev 

@xmnlab 

Please let's not have any PRs that depend on a new image. Issues a stable release and let's stick to that for a while.

**Comment by xmnlab** [comment]
ok! thanks @jreback

**Pull request merged**